generator/README.md

```diff
@@ -67,6 +67,23 @@ make WHAT=container-identity
 
 where the `WHAT` var refers to the directory being built.
 
+
+To generate the annual report template for a specific year:
+
+```bash
+make ANNUAL_REPORT=true
```

Comment:
nit/non-blocking: Could we do `make annual-report` or something like it instead? Not sure we need env vars here.

Reply:
Right now the generator uses envvars, this just documents it. Changing the generator to use flags and such is larger than the annual report generator and can be done in follow up